lib/tasks/send_survey_notifications.rake

```diff
@@ -5,17 +5,24 @@ namespace :survey_notifications do
     BATCH_SIZE = 10
     accepted_submissions = EfileSubmission.joins(:efile_submission_transitions)
                                           .where("efile_submission_transitions.to_state = 'accepted'")
+                                          .where("efile_submissions.data_source_type = 'StateFileAzIntake' OR efile_submissions.data_source_type = 'StateFileNyIntake'")
```

Comment:
the `for_state_filing` scope will cover this too

Reply:
Done